src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -288,6 +288,38 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     layout.addColorButton("Line color",
                           QColor(getSettings()->lastMessageColor.getValue()),
                           getSettings()->lastMessageColor);
+    layout.addSeperator();
+    layout.addCheckbox("Gray-out message history before connecting.",
+                       s.grayOutHistory);
+    layout.addCheckbox("Draw a line below the last message before "
+                       "connecting.",
+                       s.showBeforeConnectingIndicator);
```

Comment:
```suggestion
    layout.addCheckbox(
        "Draw a line below the last message loaded from message history.",
        s.showBeforeConnectingIndicator);
```
something like this? I just included the formatted version(s) encase it's implemented via suggestions